### PR TITLE
SAK-40128 Merge SU admin tools into one project.

### PR DESCRIPTION
**Pull request by axxter99**
1) ./tool/tool-tool/su => ./admin-su
2) pom.xlm: artifactI:admin-tool-su

### CODE REVIEW

**Comment by axxter99** [comment]
@buckett ?

File: pom.xml [hunk 1]
```diff
@@ -111,7 +111,7 @@
                 <module>syllabus</module>
                 <module>taggable</module>
                 <module>textarea</module>
-                <module>tool</module>
+                <module>admin-su</module>
```
Comment (buckett) [inline pom.xml]: Could we move this line to keep them alphabetical?

File: pom.xml [hunk 2]
```diff
@@ -194,7 +194,7 @@
...
```
Comment (buckett) [inline pom.xml]: Could we move this line to keep them alphabetical?

File: admin-su/pom.xml [hunk 3]
```diff
@@ -2,14 +2,13 @@
...
+    <artifactId>base</artifactId>
     <groupId>org.sakaiproject</groupId>
     <version>13-SNAPSHOT</version>
-    <relativePath>../../pom.xml</relativePath>
   </parent>
   <name>sakai-tool-tool-su</name>
-  <groupId>org.sakaiproject</groupId>
-  <artifactId>sakai-tool-tool-su</artifactId>
+  <groupId>org.sakaiproject.su</groupId>
+  <artifactId>admin-tool-su</artifactId>
```
Comment (buckett) [inline admin-su/pom.xml]: It's generally better if the folder name matches the `artifactId`, I know Sakai is currently bad at this but would be nice if we could start the path to making it better.

https://blog.sonatype.com/2011/01/maven-tip-project-directories-and-artifact-ids/

**Comment by buckett** [comment]
Also currently has conflicts.

**Comment by bjones86** [comment]
@buckett, are you ok with the changes here?

**Pull request merged**